integration_tests/commands/http/geo_tests.go

```diff
@@ -0,0 +1,86 @@
+package http
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
```

Comment:
Please shift to testify for assertions. You can refer to other test files for example.

Reply:
Hi @c-harish , I have addressed this comment.